main.rb

```diff
@@ -1,14 +1,56 @@
-require_relative 'classes/nameable'
-require_relative 'classes/person'
-require_relative 'classes/student'
-require_relative 'classes/teacher'
-require_relative 'classes/decorator'
-require_relative 'classes/capitalize_decorator'
-require_relative 'classes/trimmer_decorator'
-
-person = Person.new(22, 'maximilianus')
-puts person.correct_name
-puts capitalized_person = CapitalizeDecorator.new(person)
-puts capitalized_person.correct_name
-puts capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-puts capitalized_trimmed_person.correct_name
+require_relative 'app'
+
+def menu
+  system('cls')
+  system('clear')
+
+  puts ' Welcome to the School Library App'
+  puts "\nPlease choose an option by entering a number: "
+  puts '1 - List all books.'
+  puts '2 - List all people.'
+  puts '3 - Create a person,'
+  puts '4 - Create a book.'
+  puts '5 - Create a rental.'
+  puts '6 - List all rentals for a given person id.'
+  puts '7 - Exit'
+  op = gets.chomp.to_i
+  menu unless op < 8 && op.positive?
+  op
+end
+
+# def wait
+#   puts "\npress enter to continue"
+#   gets.chomp
+# end
```

Comment:
- [ ] Please remove unused or commented lines of code as they make your code base look cluttered, untidy, and difficult to read 👍